finance/process/policies.rst

```diff
@@ -144,3 +144,12 @@ Rates and charges
    (Agreed `2021-09-17`__.)
 
    __ https://docs.google.com/document/d/1OpSEtJC0jQINTB-YNexxgnHX7-J6HRSkiPKYWBSCOfg/edit#bookmark=id.gkt3aebktlyb
+
+
+
+git@github.com:astropy/astropy-project.git
+
```

Comment:
I don't understand why git is mentioned here.
```suggestion
```

Reply:
By mistake.